sdk/eventhub/event-hubs/test/public/utils/testUtils.ts

```diff
@@ -24,28 +17,16 @@ export enum EnvVarKeys {
 }
 
 export function getEnvVarValue(name: string): string | undefined {
-  if (isNode) {
-    return process.env[name];
-  } else {
-    return self.__env__[name];
-  }
-}
-
-function injectEnvironmentVariables(
-  envVars: Omit<{ [key in EnvVarKeys]: string }, EnvVarKeys.TEST_TARGET>
-): void {
-  for (const key of Object.keys(envVars) as Exclude<EnvVarKeys, EnvVarKeys.TEST_TARGET>[]) {
-    if (isNode) {
-      process.env[key] = envVars[key];
-    } else {
-      self.__env__[key] = envVars[key];
-    }
+  try {
+    return assertEnvironmentVariable(name);
+  } catch {
+    return undefined;
   }
 }
 
 export function getEnvVars(): Omit<{ [key in EnvVarKeys]: any }, EnvVarKeys.TEST_TARGET> {
   if (getEnvVarValue(EnvVarKeys.TEST_TARGET) === "mock") {
-    injectEnvironmentVariables({
+    setEnvironmentVariables({
```

Comment:
this feels a little unusual, couldn't we use `envSetupForPlayback` to set these? Does the mock target even use the recorder?

Reply:
This library doesn't use the recorder because the recorder doesn't suppoer amqp. However, I agree with you, and I think those env var helpers should be moved to test-utils instead. I can do that in another PR.